server/tests/cli/create-import-video-file-job.ts

```diff
@@ -0,0 +1,110 @@
+/* tslint:disable:no-unused-expression */
+
+import 'mocha'
+import * as chai from 'chai'
+import { VideoDetails, VideoFile } from '../../../shared/models/videos'
+const expect = chai.expect
+
+import {
+  execCLI,
+  flushTests,
+  getEnvCli,
+  getVideosList,
+  killallServers,
+  parseTorrentVideo,
+  runServer,
+  ServerInfo,
+  setAccessTokensToServers,
+  uploadVideo,
+  wait,
+  getVideo, flushAndRunMultipleServers, doubleFollow
+} from '../utils'
+
+function assertVideoProperties (video: VideoFile, resolution: number, extname: string) {
+  expect(video).to.have.nested.property('resolution.id', resolution)
+  expect(video).to.have.property('magnetUri').that.matches(new RegExp(`\\.${extname}$`))
```

Comment:
Just `indexOf(str) !== -1` should be enought. Please test the `torrentUrl` and `fileUrl` too

Reply:
Please check the `size` too